packages/webview_flutter/webview_flutter_wkwebview/test/src/web_kit_webview_widget_test.dart

```diff
@@ -212,6 +222,307 @@ void main() {
+          ));
+
+          final NSUrlRequest request =
+              verify(mockWebView.loadRequest(captureAny)).captured.single
+                  as NSUrlRequest;
+          expect(request.url, 'https://www.google.com');
+          expect(request.httpMethod, 'post');
+        });
+
+        testWidgets('POST with body', (WidgetTester tester) async {
+          await buildWidget(tester);
+
+          await testController.loadRequest(WebViewRequest(
+              uri: Uri.parse('https://www.google.com'),
+              method: WebViewRequestMethod.post,
+              body: Uint8List.fromList('Test Body'.codeUnits)));
+
+          final NSUrlRequest request =
+              verify(mockWebView.loadRequest(captureAny)).captured.single
+                  as NSUrlRequest;
+          expect(request.url, 'https://www.google.com');
+          expect(request.httpMethod, 'post');
+          expect(
+            request.httpBody,
+            Uint8List.fromList('Test Body'.codeUnits),
+          );
+        });
+      });
+
+      testWidgets('canGoBack', (WidgetTester tester) async {
+        await buildWidget(tester);
+
+        when(mockWebView.canGoBack).thenAnswer(
+          (_) => Future<bool>.value(false),
+        );
+        expect(testController.canGoBack(), completion(false));
+      });
+
+      testWidgets('canGoForward', (WidgetTester tester) async {
+        await buildWidget(tester);
+
+        when(mockWebView.canGoForward).thenAnswer(
+          (_) => Future<bool>.value(true),
+        );
+        expect(testController.canGoForward(), completion(true));
+      });
+
+      testWidgets('goBack', (WidgetTester tester) async {
+        await buildWidget(tester);
+
+        await testController.goBack();
+        verify(mockWebView.goBack());
+      });
+
+      testWidgets('goForward', (WidgetTester tester) async {
+        await buildWidget(tester);
+
+        await testController.goForward();
+        verify(mockWebView.goForward());
+      });
+
+      testWidgets('reload', (WidgetTester tester) async {
+        await buildWidget(tester);
+
+        await testController.reload();
+        verify(mockWebView.reload());
+      });
+
+      testWidgets('evaluateJavascript', (WidgetTester tester) async {
+        await buildWidget(tester);
+
+        when(mockWebView.evaluateJavaScript('runJavaScript')).thenAnswer(
+          (_) => Future<String>.value('returnString'),
+        );
+        expect(
+          testController.evaluateJavascript('runJavaScript'),
+          completion('returnString'),
+        );
+      });
+
+      testWidgets('evaluateJavascript with null return value',
+          (WidgetTester tester) async {
+        await buildWidget(tester);
+
+        when(mockWebView.evaluateJavaScript('runJavaScript')).thenAnswer(
+          (_) => Future<Object?>.value(),
+        );
+        expect(
+          testController.evaluateJavascript('runJavaScript'),
+          completion('null'),
```

Comment:
Then this becomes `'(null)'`. Probably also worth duplicating the explanation here just to make very clear that it's deliberate, and shouldn't be changed.